### PR TITLE
XVF3800 support

### PR DESCRIPTION
**Pull request by ed-xmos**
Official support for XVF3800 HW.
https://github.com/xmos/vocalfusion-rpi-setup/issues/53

Also addresses https://github.com/xmos/sw_xvf3800/issues/318

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,6 +109,18 @@ For XVF3510-UA and XVF361x-UA devices these actions will be done as well:
 
    ```./setup.sh xvf3615-int```
 
+   For XVF3800-INTDEV devices, run the installation script as follows:
```
Comment (lucianomartin) [inline README.md:112]: @mbanth, which XVF3800 devices do we need to support for the January release?

Comment (mbanth) [reply README.md:112]: We must support INT-device.  We should support INT-host if possible.  We will not support UA.

Comment (lucianomartin) [reply README.md:112]: I think you can support XVF3800, but you shouldn't list XVF3800-UA.

Comment (ed-xmos) [reply README.md:112]: I have removed the 3800 UA

**Comment by ed-xmos** [comment]
I have tested this and can confirm the the DAC does not play until the firmware runs

**Review by lucianomartin — APPROVED** [review]
Looks good!

**Review by mbanth — CHANGES_REQUESTED** [review]
IN addition to my other comments, the setup_mclk_bclk.c file needs its copyright date updated.

File: resources/clk_dac_setup/setup_io_exp_and_dac.py [hunk 2]
```diff
@@ -1,44 +1,42 @@
 #!usr/bin/python
 # Copyright (c) 2018-2022, XMOS Ltd, All rights reserved
```
[the remark this answers is not in the document]
Comment (ed-xmos) [reply resources/clk_dac_setup/setup_io_exp_and_dac.py:2]: This is going to take a while to get into my head having not had a proper NYE celebration due to illness..

**Review by mbanth — APPROVED** [review]
Looks good.

**Pull request merged**